src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.6
++++++
+* support commit and pull request git events for ACR Task source trigger
```

Comment:
This is for app service. I'd keep it to something more general such as `update ACR SDK`